Make `Task` extend `Named`, just like `Configuration` (#28282)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

. Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
. Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.

[[changes_8.8]]
== Upgrading from 8.7 and earlier

=== Deprecations

[[deprecated_namers]]
==== Deprecated `Namer` of `Task` and `Configuration`

> `Task` and `Configuration` have a link:{javadocPath}/org/gradle/api/Namer.html[`Namer`] inner class (also called `Namer`) that could be used as a common way to retrieve the name of a task or configuration. Now that these types

Reviewer: ```suggestion
`Task` and `Configuration` have a link:{javadocPath}/org/gradle/api/Namer.html[`Namer`] inner class (also called `Namer`) that can be used as a common way to retrieve the name of a task or configuration. 
```

> implement link:{javadocPath}/org/gradle/api/Named.html[`Named`], these classes are no longer necessary and have been deprecated. It will be removed in Gradle 9.0. Use link:{javadocPath}/org/gradle/api/Named.Namer.html#INSTANCE[`Named.Namer.INSTANCE`] instead.

Reviewer: ```suggestion
Now that these types implement link:{javadocPath}/org/gradle/api/Named.html[`Named`], these classes are no longer necessary and have been deprecated. 
They will be removed in Gradle 9.0. 
Use link:{javadocPath}/org/gradle/api/Named.Namer.html#INSTANCE[`Named.Namer.INSTANCE`] instead.
```

. Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
. Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.

[[changes_8.8]]
== Upgrading from 8.7 and earlier

=== Deprecations

> [[deprecated_namers]]

Author: `deprecated_namers` looks inconsistent with the `DeprecationLogger.withUpgradeGuideSection(8, "deprecated_namer")` calls (note the extra `s` at the end)

==== Deprecated `Namer` of `Task` and `Configuration`
`Task` and `Configuration` have a link:{javadocPath}/org/gradle/api/Namer.html[`Namer`] inner class (also called `Namer`) that could be used as a common way to retrieve the name of a task or configuration. Now that these types
implement link:{javadocPath}/org/gradle/api/Named.html[`Named`], these classes are no longer necessary and have been deprecated. It will be removed in Gradle 9.0. Use link:{javadocPath}/org/gradle/api/Named.Namer.html#INSTANCE[`Named.Namer.INSTANCE`] instead.

The superinterface, link:{javadocPath}/org/gradle/api/Namer.html[`Namer`], is *not* being deprecated.

> [[deprecated_access_to_conventions]]

Author: This `[[deprecated_access_to_conventions]]` may have been copied here by mistake
